part4/run.poc.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Clone oss-fuzz if not already cloned
+if [ ! -d "oss-fuzz" ]; then
+  git clone https://github.com/google/oss-fuzz.git
```

Comment:
Please use our own, pinned version, of `oss-fuzz`

```suggestion
  git clone --depth 1 https://github.com/hamzaremmal/fuzz-libpng.git -b oss-fuzz oss-fuzz
```

Reply:
Based on [this](https://github.com/hamzaremmal/fuzz-libpng/pull/1/files#r2084915034), it should be
```suggestion
  git clone --depth 1 https://github.com/hamzaremmal/fuzz-libpng.git -b vunerable/oss-fuzz oss-fuzz
```